sdk/communication/azure-communication-chat/README.md

```diff
@@ -0,0 +1,712 @@
+
+```xml
+<dependency>
+  <groupId>com.azure.android</groupId>
+  <artifactId>azure-communication-chat</artifactId>
+  <version>1.0.0-beta.1</version>
+</dependency>
+```
+
+## User and User Access Tokens
+
+User access tokens enable you to build client applications that directly authenticate to Azure Communication Services. Refer [here](https://docs.microsoft.com/azure/communication-services/quickstarts/access-tokens) to learn how to create a user and issue a User Access Token.
+
+The id for the user created above will be necessary later to add said user as a member of a new chat thread. The initiator of the create request must be in the list of members of the chat thread.
+
+## Create the AzureCommunicationChatClient
+
+```java
+import com.azure.android.communication.chat.AzureCommunicationChatServiceAsyncClient;
+import com.azure.android.core.http.HttpHeader;
+
+final String endpoint = "https://<resource>.communication.azure.com";
+final String userAccessToken = "<user_access_token>";
+
+AzureCommunicationChatServiceAsyncClient client
+    = new AzureCommunicationChatServiceAsyncClient.Builder()
+    .endpoint(endpoint)
+    .credentialInterceptor(chain -> chain.proceed(chain.request()
+        .newBuilder()
+        .header(HttpHeader.AUTHORIZATION, userAccessToken)
+        .build());
+```
+
+# Key concepts
+
+A chat conversation is represented by a chat thread. Each user in the thread is called a thread member. Thread members can chat with one another privately in a 1:1 chat or huddle up in a 1:N group chat. Users also get near real-time updates for when others are typing and when they have read the messages.
+
+Once you initialize an `AzureCommunicationChatClient` class, you can perform the following chat operations:
+
+## Thread Operations
+
+- [Create a thread](#create-a-thread)
+- [Get a thread](#get-a-thread)
+- [List threads](#list-threads)
+- [Update a thread](#update-a-thread)
+- [Delete a thread](#delete-a-thread)
+
+## Message Operations
+
+- [Send a message](#send-a-message)
+- [Get a message](#get-a-message)
+- [List messages](#list-messages)
+- [Update a message](#update-a-message)
+- [Delete a message](#delete-a-message)
+
+## Thread Member Operations
+
+- [Get thread members](#get-thread-members)
+- [Add thread members](#add-thread-members)
+- [Remove a thread member](#remove-a-thread-member)
+
+## Events Operations
+
+- [Send a typing notification](#send-a-typing-notification)
+- [Send read receipt](#send-read-receipt)
+- [Get read receipts](#get-read-receipts)
+
+# Examples
+
+## Thread Operations
+
+### Create a thread
+
+Use the `create` method to create a thread.
+
+```java
+//  The list of ChatThreadMember to be added to the thread.
+List<ChatThreadMember> members = new ArrayList<>();
+// The CommunicationUser.identifier you created before, required.
```

Comment:
This might confuse users if they picked a different programming language for getting their access token, we should make it language agnostic.
```suggestion
// The communication user id you created before, required.
```